qdrant-landing/content/documentation/qdrant-alternatives.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Qdrant vs. Pinecone
```

Comment:
Can we change the name to "Alternatives" or "Qdrant alternatives". Right now, we have our competitor mentioned in the left menu.
![image](https://github.com/qdrant/landing_page/assets/2649301/fda76a92-0fe4-425f-ad67-c83f158f371b)

Since we will extend it soon, it might be better to keep the name as general as possible.

Reply:
Qdrant vs Alternatives?

Reply:
That works for me :point_up: